api/gateway/stripe/stripe.cfc

```diff
@@ -434,16 +593,52 @@
 		<cfreturn post />
 	</cffunction>
 
+
+	<cffunction name="addBankAccount" output="false" access="private" returntype="any" hint="Add payment source fields to the request object">
+		<cfargument name="post" type="struct" required="true" />
+		<cfargument name="account" type="any" required="true" />
+
+		<cfscript>
+			post["bank_account[country]"] = arguments.account.getCountry();
+			post["bank_account[routing_number]"] = arguments.account.getRoutingNumber();
+			post["bank_account[account_number]"] = arguments.account.getAccount();
+		</cfscript>
+	
+		<cfreturn post />
+	</cffunction>
+	
+
 	<cffunction name="addToken" output="false" access="private" returntype="any" hint="Add payment source fields to the request object">
 		<cfargument name="post" type="struct" required="true" />
 		<cfargument name="account" type="any" required="true" />
 		
 		<!--- required when using as a payment source --->
-		<cfset arguments.post["customer"] = arguments.account.getID() />
```

Comment:
I think we discussed that this line needs to be reverted to use customer rather than card, right?  E.g., we will always convert a temporary card token into a customer and then use that as a payment source?  Or is this different in the latest API?